.claude/commands/prepare.md

```
@@ -0,0 +1,150 @@
+### Step 3: Run doctor
+
+```bash
+cd deploy && make doctor <topology>-<method>
+```
+
+Relay FAIL lines verbatim. Doctor is read-only — it won't break anything.
+
+### Step 4: Print deployment commands (never run them)
+
+**medium=aws:**
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)